crates/parser2/src/ast/expr.rs

```diff
@@ -927,4 +972,41 @@ mod tests {
         }
         assert_eq!(count, 3)
     }
+    #[test]
+    #[wasm_bindgen_test]
```

Comment:
```suggestion
```

I'm not sure if we need a wasm test here, is there any specific reason to run wasm test for these tests?

Reply:
Oh, sorry. I missed this is an AST test, and all AST tests use `wasm_bindgen_test`. Please ignore this comment.